x11/lightdm-kde-greeter/Makefile

```diff
@@ -0,0 +1,38 @@
+PORTNAME=	lightdm-kde-greeter
+PORTVERSION=	6.0.3
+CATEGORIES=     x11 kde
+
+MAINTAINER=     kde@FreeBSD.org
+COMMENT=        KDE login screen using the LightDM framework
+WWW=            https://invent.kde.org/plasma/lightdm-kde-greeter
+
+LIB_DEPENDS=	liblightdm-gobject-1.so:x11/lightdm \
+		libKF6NetworkManagerQt.so:net-mgmt/networkmanager-qt
+RUN_DEPENDS=	at-spi2-core>0:accessibility/at-spi2-core
+
+LICENSE=	GPLv3
+LICENSE_FILE=   ${WRKSRC}/COPYING
+
+USES=           cmake compiler:c++11-lang gettext gnome kde:6 pkgconfig qt:6 \
+		tar:xz xorg
+USE_KDE=        auth config configwidgets i18n iconthemes kcmutils package \
+		plasma-workspace \
+                ecm:build
+USE_GNOME=	gtk20
+USE_QT=         base
+USE_XORG=       x11
+
+USE_GITLAB=	yes
+GL_SITE=	https://invent.kde.org
+GL_ACCOUNT=	plasma
+GL_TAGNAME=	31d29530ad834c1bfb70ed43f9395a549fbace56
+
+CMAKE_ARGS=	-DGREETER_IMAGES_DIR="${LOCALBASE}/lib/${PORTNAME}/images" \
+		-DLIGHTDM_CONFIG_DIR=${LOCALBASE}/etc/lightdm \
+		-DDATA_INSTALL_DIR=${LOCALBASE}/share
```

Comment:
These vars should contain `${PREFIX}` instead of `${LOCALBASE}` if I understand it correctly.

While LOCALBASE==PREFIX in 99% of cases, there is a difference between these two vars. PREFIX defines where **this** port is going to be installed and where the program expects its data files to be placed. LOCALBASE is a PREFIX where other dependencies were installed.

Since we're talking about this ports' files, the PREFIX var should used.